src/menu/maybe-child-menu.tsx

```diff
@@ -8,31 +8,32 @@ LICENSE file in the root directory of this source tree.
 import * as React from 'react';
 
 import { NestedMenuContext } from './nested-menus';
-import { Popover } from '../popover/index';
+import { Popover } from '../popover';
 import type { OverrideT } from '../helpers/overrides';
 import { getOverrides, mergeOverrides } from '../helpers/overrides';
 import type { ItemT } from './types';
 
 type PropsT = {
-  children: React.ReactNode;
```

Comment:
Does React.FC imply `children` prop? Is removing this the reason for ts-expect-errors comments below?

Reply:
yes, `React.FC` will anyway add it with the same type, so it is not really needed here (adding it back btw will not change anything)

about ts-expect-errors below - that is unrelated… while technically react does allow `ReactNode` to be returned from functional components, typescript does not (there are some reasons about type-checking JSX), and that is why ts-expect-errors in code below